SEP-0002-revised.md

```diff
@@ -0,0 +1,245 @@
+# SEP-0002 - SunPy Organization Definition
+
+# Abstract
+The SunPy project has changed significantly since the first version of SEP-0002 was written in 2014.
+This SEP details a new governance structure designed to enable the project to grow, while allowing for more input from the wider SunPy community.
+
+## Detailed Description
+The primary role of the SunPy Project is to *facilitate and promote the use and development of community-led, free and open-source solar data-analysis software based on the scientific Python environment*.
+This includes but is not limited to the following tasks:
+
+* developing the sunpy core library and other affiliated packages
+* manage and protect the SunPy brand and identity
+* promote SunPy to the community
+* managing SunPy assets, i.e. GitHub organization, domain name etc.
+
+# Governance Structure
+
+The SunPy project's governance involves the following five components:
```

Comment:
I think this looks good to me.  My only concern is that this is too many layers of bureaucracy.  Perhaps that is because the board is still 5-10.  If it were guaranteed to be smaller it might feel less weighty.

Reply:
Change made here: https://github.com/sunpy/sunpy-SEP/pull/62/files#r1114322469